getsentry/sentry-wizard
# test: Add Remix e2e tests with test utils.

Adds utilities for E2E testing wizard CLI

### Non-testing updates
- This PR adds new CLI arguments to provide pre-selected project data. This is required to run tests with dummy Auth Tokens / DSNs without opening browser / signing in. Looking at the comments, I got the impression that, this feature was already planned. Please let me know if that's not the case, and I'll try to figure out another way.

### Testing utilities
The flow of the initial e2e test structure is:

- Create a new git repo in the local test application
- Run the wizard on it with pre-provided auth token / dsn.
  - Optionally check for the `stdout` for expected prompts / messages, provide conditional `stdin`
- Check for the file contents / file existences in the application after the wizard is finished. 
- Check if the project builds successfully
- Check if the project runs on production mode successfully
- Check if the project runs in development mode successfully

In the future, we can also add new utilities to check if the events/transaction from the created example page are sent successfully to Sentry
